lib/level_of_authentication.rb

```diff
@@ -1,6 +1,8 @@
 class LevelOfAuthenticationTooLowError < StandardError; end
 
 module LevelOfAuthentication
+  CURRENT_MAXIMUM_LEVEL = "level1".freeze
```

Comment:
Actually now wondering if it's better to make this read out of `config/scopes.yml ` and select highest known level from scopes?

Reply:
Could do, but this works for now and we're very unlikely to introduce further levels.

Reply:
🤷🏻 tried it here quickly: https://github.com/alphagov/govuk-account-manager-prototype/pull/805/commits/a5bf3a9dec3b0252400660bb6fe0cab6b245c921

Can drop if that doesn't work for you?